coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -69,6 +69,7 @@ class NodeCoordinatorActor(metaStore: MetaStore,
   val dsCoordNotify = new collection.mutable.HashMap[(DatasetRef, Int), List[ActorRef]]
   val clusterSelfAddr = Cluster(context.system).selfAddress
   val actorPath = clusterSelfAddr.host.getOrElse("None")
+  val defaultKeySpace = config.getString("cassandra.keyspace")
```

Comment:
Can you please remove defaultKeySpace?  1) it's not used  2) we cannot assume in coordinator that cassandra is the ColumnStore.  It might be something else.